src/include/api.h

```diff
@@ -39,9 +39,11 @@
 } while (0)
 
 /* An API call wrapped in a transaction if necessary. */
+#ifdef HAVE_TIMESTAMPS
```

Comment:
I prefer not to duplicate the `TXN_API_CALL` macros, would you consider instead adding:
```
#define WT_TXN_TIMESTAMP_FLAG_CHECK(s)                                 \
       __wt_txn_timestamp_flags((s));
#else
#define WT_TXN_TIMESTAMP_FLAG_CHECK(s)
#endif
```
And adding a line to the existing macros?

Reply:
Fixed